### PR TITLE
Update ROADMAP.md and Kit Dev docs pages

### PR DESCRIPTION
**Pull request by bmicklea**
* Updates the ROADMAP.md file with
   * Results of 0.34 and 0.4 releases
   * What's in progress now
* Removes the only-works-on-MacOS banner from the Kit Dev docs since 0.4 added support for Windows and Linux too 🎉

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -6,53 +6,59 @@ As a public community project the KitOps roadmap is always open to feedback. If
...
-* Add `kit dev` command to Kit CLI on *Windows* 💡
-* Add `kit dev` command to Kit CLI on *Linux* 💡
+## Roadmap Themes
 
-### Tutorials
+### UX Improvements
 
-#### Demos
+* [Kit CLI should be able to reuse Docker credentials](https://github.com/jozu-ai/kitops/issues/213)
+* Export [Llamafile](https://github.com/Mozilla-Ocho/llamafile?tab=readme-ov-file) from ModelKit
```
Comment (gorkem) [inline ROADMAP.md:33]: I think this is on hold right now

File: ROADMAP.md [hunk 2]
```diff
@@ -6,53 +6,59 @@ As a public community project the KitOps roadmap is always open to feedback. If
...
+* Kit Dev: Generate example code as the parameters and prompts are entered
+* Kit Dev: A way to see JSON communicated between the server and responses
+* Kit Dev: Hide parameters that are not frequently changed
```
Comment (gorkem) [inline ROADMAP.md:37]: I think we should add the pack from HF in here.

**Review by gorkem — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (bmicklea) [reply ROADMAP.md:33]: removed

File: ROADMAP.md (hunk 2 above)
Comment (bmicklea) [reply ROADMAP.md:37]: 🤦 cant' beleive I missed my favourite feature request!

**Review by gorkem — APPROVED** [review]

**Pull request merged**